setup-guide.mdx

```diff
@@ -0,0 +1,167 @@
+6. Run [`chalk init`](/cli/init) to initialize your project files. This will initialize a directory structure with an
+empty `src` directory and three files--`chalk.yaml`, `README.md`, and `requirements.txt`.
+```
+root_directory/
+├── src/
+├── chalk.yaml
+├── README.md
+└── requirements.txt
+```
+The first file, `chalk.yaml`, stores configuration information about your project. You can edit this so it
+contains the following
+   ```
+   project: {YOUR_PROJECT_NAME}
+   environments:
+     default:
+       requirements: requirements.txt
+       runtime: python310
+   ```
+
+The second file, `README.md`, contains some basic commands you can use with the Chalk CLI.
+The third file, `requirements.txt` should look something like this:
+   ```
+    requests
+    chalkpy[runtime]
+    ```
+
+   This contains your project dependencies.
+   You can add a file `.chalkignore` which will act just like a `.gitignore` file.
+7. Within your virtual environment, install the project requirements.
+```bash Terminal
+$ pip3 install -r requirements.txt
+```
+
+Now, you should have a virtual environment with all project dependencies installed and a basic project structure upon
+which we will build in the next step.
+
+## Configure Datasources
+
+Having set up a basic project directory, we want to configure the data sources from which we will load data to
+compute our features. We can do so in the Chalk dashboard.
+
+1. In the same directory from before, login or signup with Chalk directly from the command line. The
+[`chalk login`](https://docs.chalk.ai/cli/login) command will open your browser and create an
+API token for your local development, as well as redirect you to your dashboard. If you are not
+redirected you can also find the dashboard at [https://chalk.ai/projects](https://chalk.ai/projects).
+You will automatically see all of the environments in which the email you used to login has been
+provisioned as a user.
+ 2. Within the dashboard, navigate to `Data Sources` in the side bar and add all data sources that you will
+be working with here. After you have saved a data source, you can also use the `Test Data Source` button
+in the upper right hand corner of the Data Source configuration view to verify that your connection is
+valid.
+ 3. Within the working directory, we'll add a `datasources.py` file under our `src` folder to reference the
+data sources that we've added in the dashboard.
+ ```
+ root_directory/
+ ├── src/
+ │  ├── __init__.py
+ │  └── datasources.py
+ ├── chalk.yaml
+ ├── README.md
+ └── requirements.txt
+ ```
+
+Say we added a PostgreSQL data source, then our `datasources.py` might look something like this:
+```python
+pg = PostgreSQLSource(name='PG')
+```
+
+For more details on setting up data sources, [see here.](https://docs.chalk.ai/docs/integrations)
+
+##  Define Feature Sets and Resolvers
+
+Having set up our data sources, we now want to define the registry for this feature store by writing our
```

Comment:
maybe not repeat the same intro as the previous step? also, i dont think the concept of registry is user facing aka they wont know what this means